<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KerasForm</class>
 <widget class="QWidget" name="KerasForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keras Network</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QScrollArea" name="scrollAreaControl">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>514</width>
         <height>150</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>с:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxFromDraw">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxToDraw">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>по:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonPredict">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>10</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Прогноз</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxPredictCount">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>прогнозов:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>40</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>по:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxCheckToDraw">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
        <property name="value">
         <number>11</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxCheckFromDraw">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
        <property name="value">
         <number>11</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>40</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>проверять с:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxIterations">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>итераций:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxLossType">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>70</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>функция потерь:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelLossType">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>70</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>_</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxInputDraw">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>10</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>тираж на вход:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>пакетов:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxBatchSize">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>110</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonCreateModel">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>70</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Создать модель</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonStopProcess">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>110</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Остановить</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxUseGpu">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>70</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Use Gpu</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
